cypress/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
can we use same ts config as in root project?

Reply:
Root projects extends svelte config while this is just normal ts needed for cypress, I'll let it like this